src/platform-includes/distributed-tracing/how-to-use/react-native.mdx

```diff
@@ -0,0 +1,13 @@
+If you're using the current version of our React Native SDK distributed tracing will work out of the box. To get around possible [Browser CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) issues, define your `tracePropagationTargets`.
```

Comment:
```suggestion
If you're using the current version of our React Native SDK distributed tracing will work out of the box.
```

There's no CORs with React Native right?

Reply:
Yes, that's right. Thanks.